docs/technical_reference/unit_models/mvc.rst

```diff
@@ -0,0 +1,149 @@
+outlet_brine, and outlet_vapor, respectively).
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
+
+   "Overall heat transfer coefficient", ":math:`U`", "U", "None", ":math:`W/\left(m^2K\right)`"
+   "Heat transfer area", ":math:`A`", "area", "None", ":math:`m^2`"
+   "Approach temperature in", ":math:`\Delta T_{in}`", "delta_temperature_in", "None", ":math:`K`"
+   "Approach temperature out", ":math:`\Delta T_{out}`", "delta_temperature_in", "None", ":math:`K`"
+   "Log-mean temperature difference", ":math:`LMTD`", "lmtd", "None", ":math:`K`"
+   "Evaporator heat requirement", ":math:`Q_{evap}`", "heat_transfer", "None", ":math:`W`"
+
+Compressor Variables
+^^^^^^^^^^^^^^^^^^^^
+
+The condenser consists of 1 ControlVolume0DBlock. Pressure differential and work variables are constructed on the
+control volume.
+
+.. csv-table::
+   :header: "Description", "Symbol", "Variable Name", "Index", "Units"
+
+   "Pressure ratio", ":math:`PR`", "pressure_ratio", "None", ":math:`W/\left(m^2K\right)`"
+   "Isentropic efficiency", ":math:`\eta`", "efficiency", "None", ":math:`\text{dimensionless}`"
+
+Condenser Variables
+^^^^^^^^^^^^^^^^^^^
+
+The condenser consists of 1 ControlVolume0DBlock. No additional variables are constructed outside of those on the
+control volume.
+
+.. _MVC_equations:
+
+Equations
+-----------
+
+Evaporator Equations
+^^^^^^^^^^^^^^^^^^^^
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Mass balance", ":math:`\dot{m}_{feed} = \dot{m}_{vapor}^{vap}+\dot{m}_{brine}^{liq}`"
+   "Energy balance", ":math:`H_{vapor}^{vap}+H_{brine}^{vap}-H_{feed} = Q_{evap}`"
+   "Vapor temperature", ":math:`T_{vapor} = T_{brine}`"
+   "Log-mean temperature difference (Chen, 1987)", ":math:`LMTD = \left(\frac{1}{2}\left(\Delta T_{in}+\Delta T_{out}\right)\Delta T_{in}\Delta T_{out}\right)^\frac{1}{3}`"
+   "Evaporator heat requirement", ":math:`Q_{evap} = UA(LMTD)`"
+   "Approach temperature in*", ":math:`\Delta T_{in} = T_{condenser,in}-T_{brine}`"
+   "Approach temperature out*", ":math:`\Delta T_{out} = T_{condenser,out}-T_{brine}`"
+   "Heat transfer balance*", ":math:`Q_{evap} = -Q_{cond}`"
+
+\*Equations are coupled with the condenser through the ``connect_to_condenser`` method.
+
+Compressor Equations
+^^^^^^^^^^^^^^^^^^^^
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Pressure ratio", ":math:`P_{out} = (PR)P_{in}`"
+   "Isentropic temperature", ":math:`T_{out}=T_{in}(PR)^{1-\frac{1}{\gamma}}`"
+   "Efficiency", ":math:`\eta\left(H_{out}-H_{in}\right) = H_{isentropic,out}-H_{in}`"
+
+Condenser Equations
+^^^^^^^^^^^^^^^^^^^
+
+The condenser performance is related through the equations denoted by the footnote in the evaporator section.
+
+.. csv-table::
+   :header: "Description", "Equation"
+
+   "Complete condensation condition", ":math:`P_{out} >= P_{sat,out}\left(T_{out}\right)`"
+
+
```

Comment:
Should we add links to the Class documentation for each of the units here?

Reply:
Probably should, I agree. It was odd to document considering the modularity of the unit.

Reply:
Addressed in [92df892](https://github.com/watertap-org/watertap/pull/963/commits/92df892b81e1519ca461994ace7ee28b4876d492)